google/resource_compute_disk_test.go

```diff
@@ -372,6 +372,36 @@ func TestAccComputeDisk_deleteDetach(t *testing.T) {
 	})
 }
 
+func TestAccComputeDisk_computeDiskUserRegex(t *testing.T) {
+
+	shouldPass := []string{
+
+		"https://www.googleapis.com/compute/v1/projects/project-id/zones/us-central1/instances/123",
+		"https://www.googleapis.com/compute/v1/projects/123123/zones/us-central1/instances/123",
+		"https://www.googleapis.com/compute/v1/projects/hashicorptest.net:project-123/zones/us-central1/instances/123",
+		"https://www.googleapis.com/compute/v1/projects/123/zones/456/instances/789",
+	}
+
+	shouldFail := []string{
+		"https://www.googleapis.com/compute/v1/projects/project#/zones/us-central1/instances/123",
+		"https://www.googleapis.com/compute/v1/projects/project/zones/us-central#/instances/123",
+		"https://www.googleapis.com/compute/v1/projects/project/zones/us-central1/instances/?",
```

Comment:
Looks right - can you add a shouldFail test for "foo.com:bar:baz", and "foo.com:"?

Reply:
Sorry, though of course those should fail, I meant as project names:
"https://www.googleapis.com/compute/v1/projects/foo.com:bar:baz/zones/us-central1/instances/123".